fuse/dl/models/backbones/backbone_resnet.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, pretrained: bool = False, in_channels: int = 3, name: str = "
             state_dict = load_state_dict_from_url(model_urls[name])
             self.load_state_dict(state_dict)
 
+        self.fc = nn.Identity()
```

Comment:
Why? where is it used?

Reply:
I do that because it is not used, there are 2 mil parameters for nothing so by doing that I delete them.